### PR TITLE
Fix CRLF issue again (#1351)

### LINKED ISSUE #1351: [🐛 BUG] <center> tag not working in Markdown files
### What went wrong? 🤔

Users can no longer use the center tag inside Markdown files in CRLF mode. This is a regression.

image

This is what can be seen in the Python console:

```
[2024-06-04 10:35:31.237][Taipy][INFO] Detected 'ascii' encoding for file 'c:\Users\jacta\OneDrive\Bureau\taipy\Code\basic-demo\gui\t.md'.
[2024-06-04 10:35:31.436][Taipy][INFO]  * Server starting on http://127.0.0.1:5000
```

### Expected Behavior

The text should be centered.

### Steps to Reproduce Issue

*t.md*:

Working in LF: 
[t.md](https://github.com/user-attachments/files/15545591/t.md)

Not working in CRLF: 
[t.md](https://github.com/user-attachments/files/15545635/t.md)

```
<center>
Hello world
</center>
```

*t.py*
```python
from taipy.gui import Gui, Markdown

md = Markdown('t.md')

Gui(md).run(title="Frontend Demo")
```

### Browsers

Chrome

### OS

Windows

### Version of Taipy

Develop 5/31/2024

### Acceptance Criteria

- [ ] Ensure new code is unit tested, and check code coverage is at least 90%.
- [ ] Create a related issue in taipy-doc for documentation and Release Notes if relevant.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Resolve #1351

### CODE REVIEW

**Review by FredLL-Avaiga — DISMISSED** [review]
Good catch

File: taipy/gui/_renderers/__init__.py [hunk 1]
```diff
@@ -100,9 +100,9 @@ def __parse_file_content(self, content):
 
     def __sanitize_content(self, content: str) -> str:
         # replace all CRLF (\r\n) with LF (\n)
```
Comment (FabienLelaquais) [inline taipy/gui/_renderers/__init__.py]: Ah ah nice catch indeed!

And to be honest, I would have written all this:
```
        # Replace all CRLF (\r\n) and CR (\r) by LF (\n)
        return re.sub(r"\r", "\n", re.sub(r"\r\n", "\n", content))
```
... and all would be good.

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
🤔

File: taipy/gui/_renderers/__init__.py (hunk 1 above)
Comment (dinhlongviolin1) [reply taipy/gui/_renderers/__init__.py]: Make sense. Tx for the suggestion!

**Review by FredLL-Avaiga — APPROVED** [review]
👍

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**